### ISSUE TITLE
NOMINATE line on geography page results as disappeared

### ISSUE THREAD

**Issue by JeffreyBLewis** [JeffreyBLewis/WebVoteView#192]
See for example:

https://voteview.polisci.ucla.edu/district/11350%20Constitution%20Hwy,%20VA%2022957

though it appears to gone for any address.

Jeff

**Comment by lukesonnet** [comment]
The data are there, but just the visual is gone so I'm not sure I know exactly what the issues is here. @aaronrudkin?

**Comment by adamboche** [comment]
I see the line in Chrome Version 54.0.2840.71 (64-bit) on Ubuntu 16.10

[image: Inline image 1]

**Comment by aaronrudkin** [comment]
I do believe this is a browser issue. The line is in the code, it's just not showing. I looked at this a few weeks ago but couldn't get a solution -- I will redouble efforts now, as this is core functionality for that page.

**Comment by aaronrudkin** [comment]
This is a browser issue. The way I did this was that the containing table cell has a fixed width, dynamic height, and it contained a block element with a relative width (n%) and relative height (100%) of the containing cell. For some reason, some browsers are no longer rendering in this way. If I set a fixed width and height in pixels for the internal element, the border displays no problem. Still investigating but I will be able to fix this today.
